src/components/POLine/FundDistribution/FundDistributionForm.js

```diff
@@ -23,30 +21,21 @@ import { Required } from '../../Utils/Validate';
 class FundDistributionForm extends Component {
   static propTypes = {
     store: PropTypes.object.isRequired,
-    dispatch: PropTypes.func.isRequired,
-    change: PropTypes.func.isRequired,
     funds: PropTypes.arrayOf(PropTypes.object),
   };
 
   addFields = (fields) => fields.push({});
 
   removeFields = (fields, index) => fields.remove(index);
 
-  updateCode = (fundId, formItemName) => {
```

Comment:
Did you intend to remove this?

Reply:
yes, it's by intention. I left a comment that changes in that components are required because `id` was removed from `fund_distribution` on the back-end